.github/ISSUE_TEMPLATE/nova_tarefa.md

```
@@ -0,0 +1,40 @@
+### Título da Tarefa
+[Nome curto e descritivo da tarefa]
+
+### Descrição
+[Descrição detalhada do que precisa ser feito]
+
+### Objetivo
+[Qual o objetivo/resultado esperado desta tarefa]
+
+### Critérios de Aceitação
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)